readme.adoc

```diff
@@ -1,66 +1,63 @@
 == About this book
 
 _Bitcoin Development Philosophy_ is a guide for new Bitcoin developers
-who already understand the basics of, for example, proof of work, block
-building, and the transaction life cycle, who want to level-up and
-gain a deeper understanding of Bitcoin's design tradeoffs and
+who already understand the basics of concepts and processes such as Proof-of-Work, block
+building, and the transaction life cycle, and who want to level up by
+gaining a deeper understanding of Bitcoin's design trade-offs and
 philosophy. It should help new developers absorb the most important
-lessons of a decade of bitcoin development and discussion, and provide
+lessons of over a decade of Bitcoin development and discussion, and provide
 a context for evaluating new ideas (good ones and bad ones!).
 
 Table of contents overview:
 
 :oldstyle: {xrefstyle}
 :xrefstyle: full
-<<decentralization>>::
-<<trustlessness>>::
-<<privacy>>::
-<<finitesupply>>::
-<<upgrading>>::
-<<adversarialthinking>>::
-<<opensource>>::
-<<scaling>>::
-<<whenshithitsthefan>>::
-<<appendixdiscussion>>::
-<<appendixbuild>>:: {empty}
+<<Decentralization>>::
```

Comment:
We shouldn't make these changes because it can break the ability for these links to work. If the idea is that they should be capitalized, on current master, when these are resolved during building; the end result is capitalized: 
<img width="300" alt="Screen Shot 2023-01-17 at 9 18 31 AM" src="https://user-images.githubusercontent.com/23396902/212922913-ec746139-3551-4aa3-b22d-998de4cbd151.png">

Reply:
I figured there was some technical reason behind it but I suggested the changes anyway, in case someone had missed it.

Reply:
Just a note here for @MartaTurchetta : If you want to change the caption of internal links, for example the one for `finitesupply`, you can write:

`<<finitesupply,Finite Supply Chapter>>`

Which will display the text "Finite Supply Chapter" as the link text but still refer to the correct anchor ID `finitesupply`.

Also as @jarolrod mentioned, throughout the book the captions of internal links can be modified centrally, by setting a single variable. We can chose between a few pre-defined formats listed in https://docs.asciidoctor.org/asciidoc/latest/macros/xref-text-and-style/#cross-reference-styles. It's also quite flexible how the word "Chapter", "Figure", "Section", etc can be changed for anything we want (or remove it completely).

@MartaTurchetta how would you prefer cross references to be displayed in text?

Reply:
@kallerosenbaum thank you very much for your notes, very helpful and appreciated!

Re cross references, it depends on the context. In the present example, we are introducing the table of contents and this should be as complete as possible, i.e. mentioning number and title of each chapter and appendix with :xrefstyle: full. Whenever there are cross references within the text itself instead, I would keep the information as concise as possible, thus only mentioning the number of section/chapter/appendix/figure with :xrefstyle: short (I find it easier to orient myself with consequential numbers rather than titles, but if you prefer to use words :xrefstyle: basic is also fine). What do you think about it?

Reply:
Ok, sounds good to me. Let's use `:xrefstyle: short` (but locally override it when needed). Fixed in PR #66

Reply:
Ok, should I take care of this and check that this is the format we always use within the text?